ceno_zkvm/src/scheme/verifier.rs

```diff
@@ -61,10 +61,11 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> ZKVMVerifier<E, PCS>
         does_halt: bool,
     ) -> Result<bool, ZKVMError> {
         // require ecall/halt proof to exist, depending whether we expect a halt.
+        // TODO: make it less adhoc
```

Comment:
Especially once we support more ecalls.  Or perhaps we should introduce a specific 'halt-successfully' introduction.